platform_plugin_ontask/static/html/ontask.html

```diff
@@ -1,3 +1,39 @@
 <div class="ontask-instructor-wrapper">
-    <iframe class="ontask-instructor-iframe" src="{{ONTASK_URL}}" frameborder="0"></iframe>
+  {% if workflow_id %}
+  <div class="upload-dataframe">
+    <button
+      id="upload-dataframe"
+      data-course-id="{{ course_id }}"
+      class="btn btn-primary upload-dataframe-button"
+    >
+      Load data
+    </button>
```

Comment:
Maybe we can add instructions so the instructor knows what will happen when pressing the button. Also, mention that to view the data table the user must log in first.